assets/javascripts/discourse/controllers/user-activity-reactions.js

```diff
@@ -61,18 +63,18 @@ export default Controller.extend({
     CustomReaction.findReactions(this.reactionsUrl, this.username, opts)
       .then((newReactionUsers) => {
         reactionUsers.addObjects(newReactionUsers);
-        this._updateBeforeIds(newReactionUsers);
+        this.#updateBeforeIds(newReactionUsers);
         if (newReactionUsers.length === 0) {
-          this.set("canLoadMore", false);
+          this.canLoadMore = false;
         }
       })
       .finally(() => {
-        this.set("loading", false);
+        this.loading = false;
       });
-  },
+  }
 
   @observes("canLoadMore")
```

Comment:
Can you confirm that this is working correctly? I'm not sure whether `@observes` works under octane `@tracked`/autotracking

Reply:
Yup it wasn't working correctly, I've removed it in the latest commit and am updating `showFooter` differently, thanks!